lib/db/database_anonymizer.rb

```diff
@@ -217,6 +218,20 @@ def anonymize_case_sar_offender(kase)
   end
   # rubocop:enable Metrics/CyclomaticComplexity
 
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def anonymize_case_sar_offendercomplaint(kase)
+    anonymize_case_sar_offender(kase)
+    kase.ico_contact_name = Faker::Name.name unless kase.ico_contact_name.blank?
+    kase.ico_contact_email = Faker::Internet.email(name:kase.name) unless kase.ico_contact_email.blank?
```

Comment:
Does this mean that the non-anonymised case name is used in the email? We should use a fake name if this is the case 👍

Reply:
@willmcb , the kase.name has been anonymized in the previous steps `anonymize_cases()` (this fun will be applied to all case types),   the way here for generating email by using name is just to make the email more sensible in a way

Reply:
Brill I thought that would be the case but best to confirm with this 👍